### PR TITLE
Fix battery status manual tests

### PR DESCRIPTION
**Pull request by zqzhang**

### CODE REVIEW

**Comment by tobie** [comment]
Reviewers for this pull request are: @w3c/battery-status-reviewers.

File: battery-status/battery-charging-manual.html [hunk 1]
```diff
@@ -31,10 +31,10 @@
   navigator.getBattery().then(function (battery) {
     t.step(function () {
       assert_true(battery.charging, 'charging must be set to true');
-      assert_less_than(battery.chargingTime, Infinity, 'chargingTime must be less than Infinity');
+      assert_less_than_equal(battery.chargingTime, Infinity, 'chargingTime must be less than or equal to Infinity');
```
Comment (dontcallmedom) [inline battery-status/battery-charging-manual.html]: While I agree this is correct, I'm not sure there is much point to that assert — `x ≤ ∞` is always true.

Comment (zqzhang) [inline battery-status/battery-charging-manual.html]: Then we can remove this checkpoint, OK?

Comment (dontcallmedom) [inline battery-status/battery-charging-manual.html]: +1

**Pull request merged**